### DISCUSSION TITLE
extrude below minimum temp

### DISCUSSION THREAD

**Discussion by tavalin** [bassamanator/Sovol-SV06-firmware#56] [Q&A] [answered]
I'm new to klipper and have followed the instructions here for the sv06plus. I've updated the start and end gcode in the sovol cura slicer and resliced the benchy but when attempting to print, it warms up the bed and hotend and then homes, whilst doing this the hotend temp drops and I get the error message in the title. 

Has anyone else experienced this or know what I'm doing wrong?

Thanks

**Comment by bassamanator** [comment]
Can you take a picture of mainsail/fluidd showing the error?

**Comment by tavalin** [comment]
Hi @bassamanator, picture attached. Thanks
![Screenshot_20230522_073057_Chrome](https://github.com/bassamanator/Sovol-SV06-firmware/assets/1771518/a43436a7-c0cb-4e33-9e23-96338988e30c)

**Comment by bassamanator** [comment]
Are you sure both print temp and initial print temp have been set correctly?

Attach gcode here, i'll see if I can print on my sv06.

**Comment by tavalin** [comment, marked as answer]
@bassamanator I've made some progress. There were 2 things that I've discovered were wrong.  

1. You linked to how to update the slicer gcode but it looks like I used the prusa/superslice syntax rather than the Cura syntax - this meant it didn't pass through the temperatures to the gcode
2. The parameters names (e.g. EXTRUDER_TEMP) I'd set in the slicer didn't match the parameter names in the PRINT_START macro (HOTEND) so the macro code failed

**Reply by bassamanator** [reply]
So it's working now?

**Reply by tavalin** [reply]
Yes I've just managed to print my first calibration cube since I klippered it 🙂

**Reply by bassamanator** [reply]
Great! 

Please close this discussion and consider staring this repo if you found my config useful. Thanks.
